sql/update/populate_customer_in_projects.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/*
+ * Copyright (C) 2016 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This script selects the customer id related to each project and populate the new customerid
+ * column in project table.
+ */
+
+define('DB_NAME', 'phpreport');
+define('DB_USER', 'phpreport');
+define('DB_PASSWORD', 'phpreport');
+define('DB_HOST', 'localhost');
+define('DB_PORT', 5432);
+
```

Comment:
I see you used `sql/anonymize.php` as an example, but I expected you to use `update/update-from-2.1-to-2.16.php` and the like. Take a look at those; they include these values from config.php.

Also, please move this script to that directory. I'll probably rename it as `update/update-from-2.16-to-2.17.php` at some point.